src/Bedrock.Framework/Client/Client.cs

```diff
@@ -18,6 +18,8 @@ public Client(IConnectionFactory connectionFactory, ConnectionDelegate applicati
 
         public async ValueTask<ConnectionContext> ConnectAsync(EndPoint endpoint, CancellationToken cancellationToken = default)
         {
+            // How to do pooling here. Should the factory handle it?
```

Comment:
Is this comment still valid? I see it's plugged int via an extension method on the builder. Then it's transparent.